components/mpas-ocean/src/analysis_members/Registry_time_series_stats_monthly_mean.xml

```diff
@@ -134,6 +134,8 @@
 			<var name="atmosphericPressure"/>
 			<var name="normalGMBolusVelocity"/>
 			<var name="vertGMBolusVelocityTop"/>
+			<var name="normalMLEvelocity"/>
+			<var name="vertMLEVelocityTop"/>
```

Comment:
This does not have a Registry variable. I think you mean `vertMLEBolusVelocityTop`

Reply:
thanks, it's fixed now